### DISCUSSION TITLE
Saucedacity Nightly Builds

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#59] [General]
If you are interested in trying the latest nightly version of Saucedacity, you may download builds from our CI here: https://nightly.link/tenacityteam/saucedacity/workflows/cmake_build/main. Please report any bugs you encounter with these versions.

**Note**: Currently, our macOS builds have been reported to not launch. I cannot do much on this issue because I do not have a Mac, but if anyone is interested in investigating this issue, feel free to try it.

**Comment by RockyMM** [comment]
I have a MacBook, and it seems that the application is crashing upon launch. At this point, I cannot invest time into debugging further.

**Reply by generic-pers0n** [reply]
It so happens that I was using a Mac today (it's not mine; I still can't develop Saucedacity for Macs right now) and I came up with a little hack to start Saucedacity:

1. Open a terminal
2. `cd` into Contents/MacOS
3. Run the Saucedacity executable located in that folder.

Apparently, Saucedacity is technically able to *run*, so this might suggest another cause. Apparently, we use a small wrapper on macOS. This wrapper had not been touched in 2 years (the last commit was in 2020) and still contained references to Audacity. I have since updated this wrapper to reference Saucedacity instead.

Does this mean the issue has been resolved? Probably not. I could attempt to retest if I ever get the chance again, but I don't know if I'll ever get that chance again.

**Reply by generic-pers0n** [reply]
Now that I think about it, I believe it was the wrapper that was crashing on startup. This was because it was looking for an Audacity executable inside the package when one didn't exist. Now I believe this could have been fixed, but I cannot confirm and at the same time believe otherwise.

**Reply by generic-pers0n** [reply]
@RockyMM I suggest you retry the latest nightly build of Saucedacity for macOS. I think it should work fine now. Again, I'm pretty sure it was the wrapper we had that was failing.

**Reply by RockyMM** [reply]
I will try some time next week.

**Comment by generic-pers0n** [comment]
I just realized I made a duplicate post about this 😆
